### ISSUE TITLE
Tree: Missing grouping for treeview items

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/vscode#52008]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated: MAS 1.3.1
ScreenReader Versions : NVDA2018.1.1 / JAWS18.0.5038

**Repro Steps:**
1. Launch VS Code.
2. Navigate to Activity Bar and select "Explorer"(Cntrl+Shift+E) button.
3. Navigate to "Open Editors" and "File Explorer" section treeview items using "Tab".
4. Read all the treeview items.

**Actual:**
NVDA does not read out the total no of treeview items as well as its ordering(1/total,2/total and so on) when navigated using downarrow key within items. 

**Same issue has been observed in JAWS.**

**Expected:**
NVDA should read total no of treeview items when screenreader lands onto the first treeview item as well as its ordering with each of them.

**Recommendations:**
Manage the content within <fieldset> and <legend> for grouping treeview items.
or,
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The screenreader users will not be able to know the total count of treeview items present as well as which item is screenreader reading out of the total items present. This information can be useful for navigating within large treeview items.

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment For Reference:**

![a11y_vscode_viewexplorer_screenreader_missinggrouping](https://user-images.githubusercontent.com/39204909/41461891-af346508-70ae-11e8-8fae-fc6cb0026720.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
This is a general tree issue, forwarding to @joaomoreno to figure out if this is low hanging or not

**Comment by joaomoreno** [comment]
@isidorn We are using `role=tree` and a lot of `aria` roles already in these locations... I thought this would be enough for this. Any recommendations?

**Comment by isidorn** [comment]
We are using the new tree and this shuold be fixed.
We are setting the aria-setsize aria-posinset and aria-level
